src/Microsoft.Azure.Relay/WriteMode.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Microsoft.Azure.Relay
+{
+    /// <summary>
+    /// WriteMode options for HybridConnectionStream
+    /// </summary>
+    public enum WriteMode
+    {
+        /// <summary>
+        /// Write Binary frames on the WebSocket
+        /// </summary>
+        Binary,
+        /// <summary>
+        /// Write Text Frames on the WebSocket
+        /// </summary>
+        Text
```

Comment:
Can we make Text = 0 and Binary = 1 so the enum values are the same as those two on WebSocketMessageType?

(Then we could even possibly cast from one to the other if we felt so inclined.)